js/lesson-26/part1.js

```diff
@@ -0,0 +1,37 @@
+`use strict`;
+
+const vehicle = {
+    manufacturer: `AUDI`,
+    model: `A80`,
+    yearOfManufacture: `2020`,
+    averagSpeed: 50,
+    powerReserve: 600,
+    displayAllAbout() {
+        for (let key in vehicle) {
+            if (typeof vehicle[key] !== 'function') {
+                console.log(`${key} : ${this[key]}`)
```

Comment:
Also, you can try to use `console.table`. Would be nicely formatted

Reply:
I tried it and I liked it! Thanks for the interesting solution!